### PR TITLE
DOCSP-26670: add note about env variables to authn commands page

### PR DESCRIPTION
**Pull request by jwilliams-mongo**
https://jira.mongodb.org/browse/DOCSP-26670
[Build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=637d477bed8015c14bd83004)
[Stage](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-26670/connect-atlas-cli/)

### CODE REVIEW

File: source/connect-atlas-cli.txt [hunk 1]
```diff
@@ -27,6 +27,16 @@ When you connect to an existing |service| account from the
 To create a new |service| account or onboard an existing account that
 doesn't have any {+clusters+}, see :ref:`atlas-cli-onboarding`.
 
+.. note::
+
+   If you have configured :ref:`API keys <atlas-admin-api-access>`
+   to access |service|, you can set :ref:`environment variables 
+   <atlas-cli-env-vars>` instead of authenticating using the 
+   commands listed above.
```
Comment (kanchana-mongodb) [inline source/connect-atlas-cli.txt:35]: Question: Would this be better under the Use Case column in the table itself?

File: source/connect-atlas-cli.txt [hunk 2]
```diff
@@ -27,6 +27,16 @@ When you connect to an existing |service| account from the
...
+
+   You can run any {+atlas-cli+} command without passing arguments
+   or options that you define in environment variables.
```
Comment (kanchana-mongodb) [inline source/connect-atlas-cli.txt:38]: Suggestion for clarity:

```suggestion
   If you defined :ref:`environment variables <atlas-cli-env-vars>`, you can run any {+atlas-cli+} command 
   without passing arguments or options that you have defined in your environment variables.
```

**Review by kanchana-mongodb — APPROVED** [review]
2 suggestions, LGTM otherwise.

File: source/connect-atlas-cli.txt (hunk 2 above)
Comment (jwilliams-mongo) [reply source/connect-atlas-cli.txt]: I had pretty much this exact text before I revised it for concision 🤣 will go with your suggestion

File: source/connect-atlas-cli.txt (hunk 1 above)
Comment (jwilliams-mongo) [reply source/connect-atlas-cli.txt:35]: I'm not sure. I think the note would be compressed width-wise in such a narrow column. I think it also applies to both use cases technically. maybe @sarahsimpers can chime in here too?

Comment (sarahsimpers) [reply source/connect-atlas-cli.txt:35]: I agree with @jwilliams-mongo, it applies to both so having it at the top is best.

**Pull request closed without merging**